model.py

```diff
@@ -419,6 +419,16 @@ def library_stage(self, value):
     def pls_id(self):
         return ConfigurationSetting.for_library(Library.PLS_ID, self)
 
+    def number_of_patrons(self, _db):
```

Comment:
You can get a database connection from inside a model object method by calling `Session.object_session(self)`, so there's no need to pass this in. See `Library.set_hyperlink` for an example.